apps/meteor/app/authorization/server/constant/permissions.ts

```diff
@@ -67,6 +67,8 @@ export const permissions = [
 	{ _id: 'set-owner', roles: ['admin', 'owner'] },
 	{ _id: 'send-many-messages', roles: ['admin', 'bot', 'app'] },
 	{ _id: 'set-leader', roles: ['admin', 'owner'] },
+	{ _id: 'start-discussion', roles: ['admin', 'user', 'guest', 'app'] },
+	{ _id: 'start-discussion-other-user', roles: ['admin', 'user', 'owner', 'app'] },
```

Comment:
I had to add the `start-discussion` permission to this file so that it could be restored on my new end-to-end tests (with `restorePermissionToRoles`). We're currently inserting this permission somewhere else (on `Meteor.startup`) and it was missing here
`start-discussion-other-user` is not related to my tests, but I decided to bring this here too since it's created in the same `Meteor.startup` callback (so we avoid one more tech debt)